test/lib/ufe/testDeleteCmd.py

```diff
@@ -361,5 +361,90 @@ def testDeleteRestrictionHierarchyWithChildrenOnDifferentLayer(self):
         self.assertTrue(stage.GetPrimAtPath('/TreeBase/trunk'))
         self.assertTrue(stage.GetPrimAtPath('/TreeBase/newChild'))
 
+    def testDeleteAndRemoveConnections(self):
+        '''Test deleting a prim and its connections'''
+        
+        # Load a scene with a compound.
+      
+        mayaUtils.openTestScene("MaterialX", "multiple_connections.ma" )
```

Comment:
You are using a Maya scene that was created with Maya 2024. It will not load back in Maya 2022 or 2023. You have the option to rebuild the scene using Maya 2022 so the test can run across Maya versions, or to make the test run only on Maya 2024 and later using a `@SkipIf` decorator.